docs/docs/intro-graphql.md

```diff
@@ -6,3 +6,169 @@ permalink: intro-graphql/
 
 # Introduction to Mu-Haskell for GraphQL
 
+This document will help you get started in writing your first GraphQL server using Mu-Haskell and `mu-graphql`!
+
+## Using the Stack template
+
+First of all, we've prepared a Stack template to boostrap your project easily, just run:
+
+```sh
+$ stack new your-project https://raw.githubusercontent.com/higherkindness/mu-haskell/master/templates/graphql-server.hsfiles -p "author-email:haskell.curry@47deg.com" -p "author-name:Haskell Curry"
+```
+
+After doing this you'll have a compiling project with all the dependencies in place and a simple `Main.hs` file created.
+
+## Defining your GraphQL Schema
+
+The template will generate a `schema.graphql` file with a basic "hello" query, feel free to change it with your domain types. For the sake of this intro we'll use our beloved library example from the Apollo GraphQL docs:
```

Comment:
Link to the example. I would also not use "our beloved example", since not everybody may be coming from Apollo or even know what Apollo is.